spec/support/matchers/have_value.rb

```diff
@@ -0,0 +1,15 @@
+RSpec::Matchers.define :have_value do |expected|
+  match do |variable|
+    selector_id = variable.sub('$', '#')
+    value_attribute = ParserSupport.parser.find_by_selector(selector_id)[0]
+
+    unless value_attribute.nil?
+      actual_value = value_attribute.split(':')[1].strip.sub(';','')
```

Comment:
Space missing after comma.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.